main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -8,9 +8,22 @@
     "java.lang.Runtime::maxMemory": "Env, IO",
     "java.lang.Runtime::version": "Env",
     "java.lang.System::console": "IO",
+    "java.lang.System::inheritedChannel": "Sys, IO",
+    "java.lang.System::setSecurityManager": "Env, Sys, IO",
+    "java.lang.System::getSecurityManager": "Env, IO",
     "java.lang.System::currentTimeMillis": "IO",
-    "java.lang.System::getenv": "Env, IO",
     "java.lang.System::nanoTime": "IO",
+    "java.lang.System::getProperties": "Env, IO",
+    "java.lang.System::lineSeparator": "Env, IO",
```

Comment:
Did we agree on `lineSeparator` being only `IO` since it is deterministic once the platform has been determined?

Reply:
Or even `Pure?`